modules/internal/OwnedObject.chpl

```diff
@@ -18,17 +18,22 @@
  * limitations under the License.
  */
 
-
-/*
- */
+/* */
 module OwnedObject {
   use ChapelStandard;
 
+  // Ideally, this can be marked with nodoc and the doc put at the module level
+  //   since owned isn't really a 'record' or a 'type'
+  // But that prevents us from referencing a "top-level" owned reference
+  // If we one day support :noindexentry:, that could be applied at the module
+  //   level (instead of :noindex:)
+  // And then we could do :mod:`owned <OwnedObject>`
+  // For now, `fixInternalDocs.sh` will replace `record` with `type` here
```

Comment:
If you think the info in the new Line 31 is important, I suggest spelling it out like this:

// For now, `fixInternalDocs.sh` will replace ".. record:: owned" with ".. type:: owned"

or, to fit in 79 columns,

// For now, fixInternalDocs changes ".. record:: owned" -> ".. type:: owned"

Ditto `shared`.